src/map/MapRoutePoints.js

```diff
@@ -27,11 +28,13 @@ const MapRoutePoints = ({ positions, onClick }) => {
       id,
       type: 'symbol',
       source: id,
+      paint: {
+        'text-color': ['get', 'color'],
+      },
       layout: {
-        'icon-image': 'arrow',
-        'icon-allow-overlap': true,
-        'icon-rotate': ['get', 'rotation'],
-        'icon-rotation-alignment': 'map',
+        'text-field': '▲',
```

Comment:
If we don't use the arrow anymore, we should probably remove it.

Reply:
ok

Reply:
I removed the arrow